activerecord/lib/active_record/railties/databases.rake

```diff
@@ -158,6 +158,21 @@ db_namespace = namespace :db do
     puts "Current version: #{ActiveRecord::Migrator.current_version}"
   end
 
+  namespace :version do
+    desc 'Prints the current schema version number'
+    task :number => [:environment, :load_config] do
```

Comment:
@JuanitoFatas I know this is really minor but the [Rails Coding Conventions](http://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code) say:

> Use Ruby >= 1.9 syntax for hashes. Prefer { a: :b } over { :a => :b }.

Even if the rest of the file uses the old syntax, it's better to stick with the new syntax in new code.